android/assets/jsons/Civ V - Gods & Kings/Buildings.json

```diff
@@ -1008,7 +1008,7 @@
 		"culture": 5,
 		"isWonder": true,
 		"greatPersonPoints": {"Great Artist": 2},
-		"uniques": ["Culture cost of adopting new Policies reduced by [10]%"],
+		"uniques": ["[-10]% Culture cost of adopting new Policies"],
```

Comment:
Here too, maybe "for adopting" not "of adopting"?

Reply:
IMO "of" is right.

Eh, NGram viewer:

https://books.google.com/ngrams/graph?content=cost+of%2C+cost+for&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Ccost%20of%3B%2Cc0%3B.t1%3B%2Ccost%20for%3B%2Cc0

Reply:
As it already used to be 'of' in the old unique, I'm inclined to keep using 'of'